Rego/DefenderConfig.rego

```diff
@@ -443,6 +444,40 @@ SensitiveRules[{
     Policy := input.dlp_compliance_policies[_]
     Rules.ParentPolicyName == Policy.Name
     Policy.Enabled == true
+    Policy.Mode == "Enable"
+}
+
+SensitiveRules[{
+    "Name" : Rules.Name,
+	"ParentPolicyName" : Rules.ParentPolicyName,
+	"BlockAccess" : Rules.BlockAccess,
+	"BlockAccessScope" : Rules.BlockAccessScope,
+	"NotifyUser" : Rules.NotifyUser,
+	"NotifyUserType" : Rules.NotifyUserType,
+    "ContentNames" : ContentNames
+}] {
+	Rules := input.dlp_compliance_rules[_]
+	Rules.Disabled == false
```

Comment:
_Drive by review (blame ted)_ indentation is off in line 459 & 460, I believe tab was used without soft tab being enabled leading these 2 lines to being indented base on what spacing tab is set to for the user.